specs/Deployment/DeploymentAPI.md

```diff
@@ -143,10 +152,8 @@ by using DeploymentInitializeOptions object and setting ForceDeployment option b
 this API. This option when set will shut down the application(s) associated with WinAppSDK packages,
 update the WinAppSDK packages and restart the application(s).
 
-// OPENISSUE: If this option is set when updating main package, then whether the out of process com
-server from the main package such as push Notifications needs to be explicitly restarted needs to be
-understood. If restart it is needed, then the API will handle explicitly restarting it. This is
-under investigation.
+If this option is set when updating singleton package, then the out of process com server from the
```

Comment:
The fact the Singleton has a Packaged COM OOP server is an implementation detail du jour. Better to avoid such details that can age badly.

SUGGESTION: `If this option is set any processes running with the Main and Singleton packages' identities are explicitly restarted (if necessary) after the package updates complete.`